src/main/java/org/folio/circulation/domain/policy/lostitem/LostItemPolicy.java

```diff
@@ -111,10 +121,10 @@ public ChargeableFee getActualCostFee() {
   }
 
   public boolean shouldRefundFees(DateTime lostDateTime) {
-    final DateTime now = ClockManager.getClockManager().getDateTime();
+    final Period overdueMinutesInterval = ofMinutesTillNow(lostDateTime);
```

Comment:
I know I established the convention of using `of` for these kinds of functions. I'm finding it difficult to read this name. There doesn't appear to be a good natural name under the constraints of Java, what do you think of the below suggestions?

```suggestion
    final Period overdueMinutesInterval = minutesFromDateToNow(lostDateTime);
```

```suggestion
    final Period overdueMinutesInterval = minutesFromDateTillNow(lostDateTime);
```

```suggestion
    final Period overdueMinutesInterval = minutesBetweenDateAndNow(lostDateTime);
```

Reply:
I've chosen the `minutesBetweenDateAndNow`